packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -78,7 +76,8 @@ export class Carequality {
   }
 
   /**
-   * Lists the indicated number of organizations. Mostly used for testing purposes.
+   * Lists the indicated number of organizations.
+   *
    * @param count Optional, number of organizations to fetch.
    * @param start Optional, the index of the directory to start querying from
    * @param start Optional, the OID of the organization to fetch
```

Comment:
Already there, but ideally we would list the default value on the TSDoc so the client don't need to look at the code to know what will be used if that param is omitted.